.yarn/versions/0e4a4eac.yml

```diff
@@ -0,0 +1,4 @@
+releases:
+  "@yarnpkg/cli": minor
+  "@yarnpkg/plugin-essentials": minor
```

Comment:
This is a breaking change so these need to be marked as `major`.

Reply:
Do you think all of these should be `major` changes? Technically this could also be thought as a breaking change for the CLI, although only for interactive mode, which users review manually anyway

Reply:
I marked this one as a `major` version bump. I left other ones intact. I also declined some other version bumps. I have no clue if this is correct since I don't know the inter-dependencies between these packages. Feel free to update this file yourself or let me know how it should be configured.